org.lflang/src/org/lflang/generator/python/PythonGenerator.xtend

```diff
@@ -856,13 +856,13 @@ class PythonGenerator extends CGenerator {
         from setuptools import setup, Extension
         
         linguafranca«topLevelName»module = Extension("LinguaFranca«topLevelName»",
-                                                   sources = ["«topLevelName».c", «FOR src : targetConfig.compileAdditionalSources SEPARATOR ", "» "«src»"«ENDFOR»],
-                                                   define_macros=[('MODULE_NAME', 'LinguaFranca«topLevelName»')«IF (targetConfig.threads !== 0 || (targetConfig.tracing !== null))», 
-                                                       ('NUMBER_OF_WORKERS', '«targetConfig.threads»')«ENDIF»])
+                                               sources = ["«topLevelName».c", «FOR src : targetConfig.compileAdditionalSources SEPARATOR ", "» "«FileConfig.toUnixString(src)»"«ENDFOR»],
```

Comment:
@lhstrh the string version of `FileConfig.toUnixString` is called here. Does it make sense to convert `src` to a path first? If so, would you please show me how I could do that efficiently?